src/planner/logical_operator.cpp

```diff
@@ -167,7 +167,7 @@ idx_t LogicalOperator::EstimateCardinality(ClientContext &context) {
 		max_cardinality = MaxValue(child->EstimateCardinality(context), max_cardinality);
 	}
 	has_estimated_cardinality = true;
-	return max_cardinality;
+	return estimated_cardinality = max_cardinality;
```

Comment:
Could we split this into two separate lines for clarity?

```cpp
estimated_cardinality = max_cardinality;
return estimated_cardinality;
```

Reply:
Sure